modules/installation-initializing.adoc

```diff
@@ -207,15 +207,10 @@ $ ./openshift-install create install-config --dir <installation_directory> <1>
 <1> For `<installation_directory>`, specify the directory name to store the
 files that the installation program creates.
 +
-[IMPORTANT]
-====
-Specify an empty directory. Some installation assets, like bootstrap X.509
-certificates have short expiration intervals, so you must not reuse an
-installation directory. If you want to reuse individual files from another
-cluster installation, you can copy them into your directory. However, the file
-names for the installation assets might change between releases. Use caution
-when copying installation files from an earlier {product-title} version.
-====
+When specifying the directory:
+* Verify that the directory has the `execute` permission. This permission is required to let the installer run Terraform binaries under the installation directory.
```

Comment:
```suggestion
* Verify that the directory has the `execute` permission. This permission is required to run Terraform binaries under the installation directory.
```
The [ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#l)'s guidance to avoid the use of let and allow for inanimate objects.

If it can be cut without changing the meaning too much, I think the sentence is stronger without it. 

If it is important, then I think that what you have is a solid compromise. In riffing on a few alternatives, everything I came up with added complexity.

Reply:
Done.